test_reflectable/test/exported_main_lib.dart

```diff
@@ -4,9 +4,9 @@
 
```

Comment:
Perhaps document the choice between:
- In the importing library, make an actual main calling `initializeReflectable` and the imported main
- modifying the library exporting the main.

Reply:
I added a comment which should describe these things.